### PR TITLE
Partner docs

### PR DESCRIPTION
**Pull request by B-Venkatesh7210**

### CODE REVIEW

**Comment by livingrockrises** [comment]
Please resolve conflicts. @B-Venkatesh7210 
cc @himanshugarg06

**Comment by himanshugarg06** [comment]
LGTM

File: partners/crosschain.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_label: "Cross-chain Swap API-s"
+sidebar_position: 5
+---
+
+# Cross-chain Swap API-s
+
+## **Seamless Crypto-to-Crypto Swap Integration**
+
+We've established partnerships with leading crypto-to-crypto swap service providers and aggregators to facilitate a seamless integration of swap functionalities within our platform. Through their developer integration toolkits, users can now exchange cryptocurrencies directly within any app, website, or web plugin. This collaboration enables users to swap between various crypto assets without the need for multiple transactions or dealing with the intricacies of swapping different assets on separate platforms. It simplifies the process and ensures a convenient experience for managing their crypto portfolios. Please note that compliance with relevant regulatory requirements may apply for working with some of these providers.
```
Comment (NunoFilipeSantos) [inline partners/crosschain.md:10]:
```suggestion
We've established partnerships with leading crypto-to-crypto swap service providers and aggregators to facilitate a seamless integration of swap functionalities within our platform. Through their developer integration toolkits, users can now exchange cryptocurrencies directly within any app, website, or web plugin. This collaboration enables users to swap between various crypto assets without the need for multiple transactions or dealing with the intricacies of swapping different assets on separate platforms. It simplifies the process and ensures a convenient experience for managing their crypto portfolios. Please note that compliance with relevant regulatory requirements may apply to working with some of these providers.
```

File: partners/crosschain.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Review the details of the exchange, including swap amount, destination crypto wallet address, exchange rate, as well as any transaction fees incurred. Click the Swap button and you’ll be redirected to your wallet interface.
+
+> ⚠️ If this is the user’s first time using Moonpay they will be required to go through a simple KYC process to comply with regulations.
+
+### 3. **Sign the transaction in your wallet**
+
+In your wallet application, sign the transaction to send crypto to the MoonPay wallet address to complete the exchange. Make sure that you have enough crypto in your wallet to cover any network fees.
+
+> 💡 Using **Session Keys** within your app will eliminate the need for this step, and using our **Paymaster** will eliminate the need for the user to keep blockchain’s native token to complete the swap.
```
Comment (NunoFilipeSantos) [inline partners/crosschain.md:43]:
```suggestion
> 💡 Using **Session Keys** within your app will eliminate the need for this step, and using our **Paymaster** will eliminate the need for the user to keep the blockchain’s native token to complete the swap.
```

File: partners/socialLogins.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+As you might already know, there are two types of accounts i.e. Smart Accounts and Externally Owned Accounts (EOAs). A lot of web3 users have EOAs that they manage using their Metamask and other similar self-custodial wallet services.
+
+Similar to EOAs, Smart Accounts also have a signer vis-a-vis a validation logic which can be set to an ECDSA signature. These ECDSA signatures can be generated using an associated private key. Developers have the flexibility to choose a singer to manage these private keys for the user. Services like Web3Auth, Particle, etc provide creative and more user friendly ways like Passkeys, Social Login and Emails to use the said private keys which are secured using different security methods like MPC, DKG, etc.
+
+We've rounded up the best social login options out there, all in one spot for you. The Biconomy stack works smoothly with these logins, supporting easy integration. When developers pick wallets for their apps, they should think about things like security, cost, user experience, flexibility, privacy, and recoverability. Check out the table below to get a quick look at these factors.
```
Comment (NunoFilipeSantos) [inline partners/socialLogins.md:14]:
```suggestion
# Social Login - Signers for your SA

Every account generated on a blockchain has a logic that allows us to cryptographically sign transactions using our private keys. Authentication services like Particle, Web3Auth, and wallet applications such as Metamask provide us with easier options to manage private keys and initiate transactions.

As you might already know, there are two types of accounts i.e. Smart Accounts and Externally Owned Accounts (EOAs). A lot of web3 users have EOAs that they manage using their Metamask and other similar self-custodial wallet services.

Similar to EOAs, Smart Accounts also have a signer vis-a-vis a validation logic which can be set to an ECDSA signature. These ECDSA signatures can be generated using an associated private key. Developers have the flexibility to choose a singer to manage these private keys for the user. Services like Web3Auth, Particle, etc provide creative and more user-friendly ways like Passkeys, Social Login, and Emails to use the said private keys which are secured using different security methods like MPC, DKG, etc.

We've rounded up the best social login options out there, all in one spot for you. The Biconomy stack works smoothly with these logins, supporting easy integration. When developers pick wallets for their apps, they should think about things like security, cost, user experience, flexibility, privacy, and recoverability. Check out the table below to get a quick look at these factors.
```

File: partners/walletUiKit.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+**What do dApps get by integrating with the Obvious Embed kit?**
+
+- Obvious embed kit helps developers to build for both EOA & SCW by abstracting out complexities around gas fees.
+- Standard signer & provider interface for EOA & SCW
+- UI Kit for showing portfolio across multiple chains, including tokens, NFTs & transaction history
+- UI Kit for 1-click swapping & bridging of tokens
+- Customizations (colors & styles) that can be plugged into your dApp, as a popup or a separate tab, with minimal effort.
+- [Obvious](https://www.obvious.technology/) as a wallet partner, built on top of Biconomy Account Abstraction SDK, users of your dApp can take their credentials and log in to Obvious Wallet for a complete wallet experience.
+
+The pictures below give you a peek into what the Obvious Embed UI looks like and the cool features it comes with.
```
Comment (NunoFilipeSantos) [inline partners/walletUiKit.md:19]:
```suggestion
Developers can now integrate Biconomy’s account abstraction-based wallet in their apps in just 5 minutes and a few lines of code. Our partners at Obvious have built out a UI kit on our existing infrastructure that removes the need for developers to build an in-app wallet UI experience for the user. The developers can now use the Obvious Embed for a smooth in-app Smart Account experience.

**What do dApps get by integrating with the Obvious Embed kit?**

- Obvious embed kit helps developers to build for both EOA & SCW by abstracting out complexities around gas fees.
- Standard signer & provider interface for EOA & SCW
- UI Kit for showing portfolio across multiple chains, including tokens, NFTs & transaction history
- UI Kit for 1-click swapping & bridging of tokens
- Customizations (colors & styles) that can be plugged into your dApp, as a popup or a separate tab, with minimal effort.
- [Obvious](https://www.obvious.technology/) as a wallet partner, built on top of Biconomy Account Abstraction SDK, users of your dApp can take their credentials and log in to Obvious Wallet for a complete wallet experience.

The pictures below give you a peek into what the Obvious Embed UI looks like and the cool features it comes with.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
Small corrections.
@himanshugarg06 please also take a look before merging.

**Review by himanshugarg06 — APPROVED** [review]

**Pull request merged**